modules/installation-osp-kuryr-ingress-scaling.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-kuryr-octavia-scale_{context}"]
+= Scaling for ingress traffic by using {rh-openstack} Octavia
+
+You can use Octavia load balancers to scale Ingress controllers on clusters that use Kuryr.
+
+.Prerequisites
+
+* Your {product-title} cluster uses Kuryr.
+
+* Octavia is available on your {rh-openstack} deployment.
+
+.Procedure
+
+. To copy the current internal router service, on a command line, enter:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress get svc router-internal-default -o yaml > external_router.yaml
+----
+
+. In the file `external_router.yaml`, change the values of `metadata.name` and `spec.type` to
+`LoadBalancer`.
++
+[source,yaml]
+.Example router file
+----
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    ingresscontroller.operator.openshift.io/owning-ingresscontroller: default
+  name: router-external-default <1>
+  namespace: openshift-ingress
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: http
+  - name: https
+    port: 443
+    protocol: TCP
+    targetPort: https
+  - name: metrics
+    port: 1936
+    protocol: TCP
+    targetPort: 1936
+  selector:
+    ingresscontroller.operator.openshift.io/deployment-ingresscontroller: default
+  sessionAffinity: None
+  type: LoadBalancer <2>
+----
+<1> Ensure that this value is descriptive, like `router-external-default`.
+<2> Ensure that this value is `LoadBalancer`.
+
+[NOTE]
+====
+You can delete timestamps and other information that is irrelevant to load balancing.
+====
+
+. From a command line, create a service from the `external_router.yaml` file:
++
+[source,terminal]
+----
+$ oc apply -f external_router.yaml
+----
+
+. Verify that the service's external IP address is the same as the one that is associated with the load balancer:
```

Comment:
To avoid the apostrophe (ISG), maybe:

. Verify that the external IP address of the service is the same as the one that is associated with the load balancer:

.. On a command line, retrieve the external IP address of the service: